index.js

```diff
@@ -1,10 +1,56 @@
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+
 const { Input } = require('./lib/Input.js');
-const { Note } = require('./lib/Note.js');
+const Note = require('./lib/Note.js');
+
+// make a new input class with command line arguements parsed in
+const noteObject = new Input(process.argv);
+console.log(noteObject);
+
+if(Input.valid(noteObject)) Note.execute(noteObject);
 
+// not sure what this does
+mongoose.connect('mongodb://localhost:27017/play', {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+});
 
-const noteObject = new Input;
-const noteObjectParse = noteObject.parse(process.argv);
+app.use(express.json());
 
-if(Input.valid(noteObjectParse)) console.log('my note', Note.execute(noteObjectParse));
+// Note.execute(noteObject);
 
-console.log(noteObjectParse);
+// need to make all these paths somehow passed from command line argv
+// make a note
+app.post('/notes', (req, res) => {
```

Comment:
no need to make an express application. Just needed to make a command line application:

```js
const mongoose = require('mongoose');
const Input = require('./lib/Input')
const Note = require('./lib/models/Note');

mongoose.connect('mongodb://localhost:27017/notes', {
  useNewUrlParser: true,
  useUnifiedTopology: true
});

const input = new Input(process.argv);

if(input.valid()) {
  Note.execute(input)
    .then(() => mongoose.connection.close());
} else {
  console.log('Something went wrong');
}
```